mirrord/config/src/feature/fs/advanced.rs

```diff
@@ -17,9 +17,9 @@ use crate::{
 
 /// Allows the user to specify the default behavior for file operations:
 ///
-/// 1. `"read"` - Read from the remote file system (default)
+/// 1. `"read"`/`true` - Read from the remote file system (default)
```

Comment:
```suggestion
/// 1. `"read"` or `true` - Read from the remote file system (default)
```

I think it's a bit more readable like this.